### PR TITLE
Add Pyth EMA

### PR DESCRIPTION
**Pull request by alittlezz**

### CODE REVIEW

File: configs/mainnet/hubble.json [hunk 1]
```diff
@@ -279,5 +279,76 @@
...
+    "oracle_mapping": "3vxLXJqLqF3JG5TCbYycbKWRBbCJQLxQmBGCkyqEEefL",
+    "oracle_type": "PythEMA"
+  },
+  "63": {
+    "label": "SLND/USD",
+    "oracle_mapping": "HkGEau5xY1e8REXUFbwvWWvyJGywkgiAZZFpryyraWqJ",
+    "oracle_type": "PythEMA"
+  },
+  "64": {
+    "label": "DAI/USD",
```
Comment (y2kappa) [inline configs/mainnet/hubble.json:349]: "DAI/USD EMA"

File: Makefile [hunk 2]
```diff
@@ -128,13 +128,19 @@ test-validator:
...
+## $ make clone-mainnet-to-local-validator
 clone-mainnet-to-local-validator: $(SCOPE_CLI)
 >@ export ORACLE_PUBKEYS="${shell CLUSTER=mainnet make print-pubkeys}"
-> solana-test-validator -r --url mainnet-beta --clone $$ORACLE_PUBKEYS
+> solana-test-validator -r --url $(URL) --clone $$ORACLE_PUBKEYS
 
 clone-devnet-to-local-validator:
 >@ export ORACLE_PUBKEYS="${shell CLUSTER=devnet make print-pubkeys}"
-> solana-test-validator -r --url devnet --clone $$ORACLE_PUBKEYS
+> solana-test-validator -r --url $(URL) --clone $$ORACLE_PUBKEYS
```
Comment (oeble) [inline Makefile:143]: This does not work anymore if you forget to set the URL as the URL is set by default to "localnet".

File: configs/devnet/hubble.json [hunk 3]
```diff
@@ -276,5 +276,87 @@
     "label": "JITOSOL/SOL",
     "oracle_mapping": "6NjY29fsq34pTqEmu2CXqGijsGLDSPdHqEyJ3fBkMxtB",
     "oracle_type": "SplStake"
+  },
+  "52": {
+    "label": "SOL/USD EMA",
```
Comment (oeble) [inline configs/devnet/hubble.json:281]: Suggestion: put `EMA` in front for better readability (same as switchboard twaps)

File: programs/scope/src/oracles/pyth_ema.rs [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    let pyth_ema_price = if cfg!(feature = "skip_price_validation") {
+        // Don't validate price in tests
+        pyth_raw.get_ema_price_unchecked()
+    } else if let Some(pyth_ema_price) = pyth_raw.get_ema_price() {
+        // Or use the current valid price if available
+        pyth_ema_price
+    } else {
+        msg!("No valid EMA price in pyth account {}", price_info.key);
+        return err!(ScopeError::PriceNotValid);
+    };
```
Comment (oeble) [inline programs/scope/src/oracles/pyth_ema.rs:36]: Should we add some ts tests?

**Review by oeble — COMMENTED** [review]
Good job

**Pull request merged**